packages/firebase_app_check/firebase_app_check/android/build.gradle

```diff
@@ -53,6 +53,7 @@ android {
     // TODO: Since App Check is a beta SDK it's not available in the Firebase Android BoM so we need to specify an exact version here.
     implementation 'com.google.firebase:firebase-appcheck-safetynet:16.0.2'
     implementation 'com.google.firebase:firebase-appcheck-debug:16.0.2'
```

Comment:
```suggestion
    // TODO: Since App Check is a beta SDK it's not available in the Firebase Android BoM so we need to specify an exact version here.
    implementation 'com.google.firebase:firebase-appcheck-safetynet:16.0.2'
    implementation 'com.google.firebase:firebase-appcheck-debug:16.0.2'
```

Is this still needed with the TODO, are the versions in the BoM now - can address separate PR if need be, just checking :D

Reply:
Looks like they're part of the BOM now as of `30.0.0`. I'll remove them from PR.
<img width="866" alt="Screenshot 2022-10-24 at 14 50 44" src="https://user-images.githubusercontent.com/16018629/197541827-7b435171-d635-4805-ba55-ee4c5b6e5bfc.png">

Reply:
Compared to the proceeding BOM before `30.0.0`:
<img width="842" alt="Screenshot 2022-10-24 at 14 51 30" src="https://user-images.githubusercontent.com/16018629/197541993-c9507eba-e22d-4115-9d81-78d64c6150c1.png">

Reply:
I guess they are still needed as build fails due to missing symbols from those packages: https://github.com/firebase/flutterfire/actions/runs/3313899595/jobs/5472489288